### ISSUE TITLE
Find hosting for Sealing App

### ISSUE THREAD

**Issue by jeremylang** [codeforboston/docassemble-juvenilesealing#101]
## Goal
- Find a home for the Juvenile Sealing App that can support it as mission-critical software so that we can have conversations with government partners at probation and courts.
- Find a partner who can provide "what happens when it goes boom in the middle of the night" level of support for after we build and deploy something.

## Tasks
- [x] Get a friendly URL, don't want it to say "dev"
- [x] Make sure we can continue to push updates after launch
- [x] Make sure Pauline has an opportunity to do a final review before launch
- [x] Ensure the Clean Slate team can continue to upgrade the app after it has gone live
- [ ] Determine if GBLS is the right home for the Sealing app

## Notes
- Code for Boston is not setup to be a support organization for critical applications. Can't be held to any kind of SLA for things a partner needs in order to do business.

**Comment by jeremylang** [comment]
#100 was split out into multiple follow up issues. We decided on 7/14 that none of the issues are show stoppers, though we'd love to work on them as soon as we can.

**Comment by jeremylang** [comment]
Follow up email send to GBLS regarding hosting.

**Comment by jeremylang** [comment]
The new tech bill (https://github.com/codeforboston/clean-slate/issues/164) might mean a questionable future for the Sealing App. However, we've decided that we still want to move forward with launching it.

I've reached out to Pauline about moving it forward within GBLS. She'll get in contact with their head of IT and see if we can move this forward. Might take a little bit as a lot of people are out on vacation this month.
